app/javascript/utils/form/final_form/Field.js

```diff
@@ -65,25 +70,22 @@ export const InputField = ({ fieldName, label, blockNote, validation, placeholde
   </Field>
 )
 
-export const CurrencyField = ({ fieldName, validation, id, label, placeholder, maxLength, disabled }) => (
-  <Field name={fieldName} validate={validation} component='input' format={formUtils.formatPrice} parse={identity} formatOnBlur>
+export const CurrencyField = ({ fieldName, validation, id, label, placeholder, maxLength, disabled, isDirty = true }) => (
+  <Field name={fieldName} validate={validation} component='input' format={formUtils.formatPrice} parse={(value, name) => identity(value, name, true)} formatOnBlur={isDirty}>
```

Comment:
Can you think of a way to make the identity fn a little clearer here rather than just passing "true" as the 3rd param? I'm not sure I can but I'm just curious

Reply:
unfortunately I cannot see another way except for passing a string and checking that value but the boolean is more performant